src/ai/backend/client/output/json.py

```diff
@@ -122,6 +123,29 @@ def print_paginated_list(
             **_json_opts,
         ))
 
+    def print_mutation_result(
+        self,
+        item: Mapping[str, Any],
+        item_name: Optional[str] = None,
+        add_info: Mapping = {},
```

Comment:
Rename `add_info` to `extra_info` or `additional_info` to make "add" to stand out as an adjective instead of a verb.